tests/test_quantum_gradients.py

```diff
@@ -281,6 +281,48 @@ def circuit(x, y):
         self.assertAllAlmostEqual(grad_A, grad_F, delta=self.tol)
         self.assertAllAlmostEqual(grad_A2, grad_F, delta=self.tol)
 
+    def test_CVOperation_with_heisenberg_and_no_params(self):
+        """An integration test for CV gates that support analytic differentiation
+        if succeeding the gate to be differentiated, but cannot be differentiated
+        themselves (for example, they may be Gaussian but accept no parameters).
+
+        This ensures that, assuming their _heisenberg_rep is defined, the quantum
+        gradient analytic method can still be used, and returns the correct result."""
+        self.logTestName()
+
+        for cls in qml.ops.cv.all_ops:
```

Comment:
Does it make sense to use `cls` as the variable name here? It would be like using a variable `self` outside of a class instance. Not forbidden, but at the very least, it throws off the code highlighting

Reply:
Agreed, this is copied from previous tests in the test suite, and should become a separate issue/PR.

Reply:
This is part of issue #99